psc-docs/Ctags.hs

```diff
@@ -1,5 +1,6 @@
 module Ctags (dumpCtags) where
 
+import Prelude
```

Comment:
I'd rather keep these changes out of the repo if we can. This should be fixed in intero if it's really a bug.